docs/sources/developers/plugins/error-handling.md

```diff
@@ -0,0 +1,61 @@
++++
+title = "Error handling"
+type = "docs"
++++
+
+# Error handling
+
+This guide explains how to handle errors in plugins.
+
+## Provide usable defaults
+
+Allow the user to learn your plugin in small steps. By providing a usable default configuration, the user can get started right away, and you can avoid unnecessary error messages early on.
```

Comment:
```suggestion
Allow the user to learn your plugin in small steps. Provide a useful default configuration so that: 
- The user can get started right away. 
- You can avoid unnecessary error messages.
```